tests/mail/tests.py

```diff
@@ -132,6 +132,22 @@ def test_cc(self):
             ['to@example.com', 'other@example.com', 'cc@example.com', 'cc.other@example.com', 'bcc@example.com']
         )
 
+    def test_cc_headers(self):
+        email = EmailMessage(
+            'Subject', 'Content', 'bounce@example.com', ['to@example.com'],
+            cc=['foo@example.com'], headers={'Cc': 'override@example.com'},
+        )
+        message = email.message()
+        self.assertEqual(message['Cc'], 'override@example.com')
+
+    def test_cc_headers_only(self):
```

Comment:
This doesn't seem to work as a regression test (additional test coverage is fine, but shouldn't be mixed with bug fixes as it makes it difficult to see what behavior is fixed).